css/style.css

```diff
@@ -374,6 +377,32 @@ video {
     margin-bottom: 90px;
   }
 
+  .carrousel-container {
+    margin-bottom: 33px;
+  }
+
+  .carrousel__arrow {
+    height: 62px;
+    width: 40px;
+  }
+
+  .carrousel__arrow--right {
+    right: 32px;
+  }
+
+  .carrousel__arrow--left {
+    left: 32px;
+  }
+
+  .pointer-list {
```

Comment:
BEM!!!!!!

Reply:
@camigomez35 i believe this name was pre-existent, also what BEM could be applied to it if it is a block on the block->element->modifier hierarchy?